python/ray/exceptions.py

```diff
@@ -204,6 +204,14 @@ def __str__(self):
                 "Check python-core-worker-*.log files for more information.")
 
 
+class FunctionLoadingError(RayError):
+    """Indicates that the function or class load failed."""
+
+    def __str__(self):
+        return ("Failed to load function or class from local. And this node "
+                "don't allow execute task from dynamic pickled function.")
```

Comment:
Does each node use a different code loading strategy? from local or not from local.

Reply:
Yep, we can allow different code load mode in one cluster.